dss/util/auth/auth0.py

```diff
@@ -24,21 +24,41 @@ def security_flow(self, *args, **kwargs):
         kwargs of this method, and used to call the correct method.
         """
         #  TODO add some type of jwt inspection
-        requested_method = kwargs.get('auth_method').lower()
-        if requested_method is None or requested_method not in self.valid_methods.keys():
-            err = f'Unable to locate auth_method {requested_method} for request, valid methods are: '
-            err += f'{", ".join(self.vaid_methods)}'
-            raise DSSException(500, err)
+
+        # Get name of method to use
+        if 'auth_method' in kwargs:
+            method = kwargs['auth_method']
+        elif len(args) > 0:
+            method = args[0]
+            kwargs['auth_method'] = method
         else:
-            executed_method = self.valid_methods[requested_method]
-            executed_method(*args, **kwargs)
+            raise RuntimeError("Error: invalid arguments passed to Auth0 security_flow() method")
```

Comment:
perhaps pass back the method that was used in decorator, would allow operator an easier time to fix.